src/content/content.service.ts

```
@@ -12,308 +12,336 @@ const crypto = require('crypto');
+    ).map((id) => {
+      return arrayOfObj.find((obj) => obj.order_id === id);
+    });
+
+    if (body.fields) {
+      console.log('body.fields', body.fields);
+      const keysToKeep = body.fields;
+
+      const result = uniqueObjects.map((obj) => {
+        const newObj = {};
+        keysToKeep.forEach((key) => {
+          if (obj.hasOwnProperty(key)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)